packages/calcite-components/src/components/pagination/pagination.scss

```diff
@@ -1,42 +1,86 @@
+ * @prop --calcite-pagination-border-color-selected: Specifies the border color of the pagination items when selected.
+ * @prop --calcite-pagination-text-color-hover: Specifies the text color of the pagination items items while hovered.
+ * @prop --calcite-pagination-text-color-selected: Specifies the text color of the pagination items when selected.
+ * @prop --calcite-pagination-text-color: Specifies the text color of the pagination items.
+ * @prop --calcite-pagination-arrow-background-color-active: Specifies the background color of the arrow items while active.
+ * @prop --calcite-pagination-arrow-background-color-hover: Specifies the background color of the arrow items while hovered.
+ * @prop --calcite-pagination-arrow-background-color: Specifies the background color of the arrow items.
+ * @prop --calcite-pagination-arrow-text-color-hover: Specifies the text color of the arrow items while hovered.
+ * @prop --calcite-pagination-arrow-text-color: Specifies the text color of the arrow items.
+ *
+ */
+
+/**
+  * Local props
+  * These properties are intended for internal component use only. It is not recommended that these properties be overwritten.
+  *
+  * --calcite-internal-pagination-item-size
+  * --calcite-internal-pagination-item-space
+*/
+
 :host {
   @apply flex;
   writing-mode: horizontal-tb;
+  --calcite-pagination-background-color-active: transparent;
+  --calcite-pagination-background-color-hover: transparent;
+  --calcite-pagination-background-color: transparent;
+  --calcite-pagination-border-color-hover: var(--calcite-color-border-2);
+  --calcite-pagination-border-color-selected: var(--calcite-color-brand);
+  --calcite-pagination-text-color-hover: var(--calcite-color-text-1);
+  --calcite-pagination-text-color-selected: var(--calcite-color-text-1);
+  --calcite-pagination-text-color: var(--calcite-color-text-3);
+  --calcite-pagination-arrow-background-color-active: var(--calcite-color-foreground-3);
+  --calcite-pagination-arrow-background-color-hover: var(--calcite-color-foreground-2);
+  --calcite-pagination-arrow-background-color: var(--calcite-pagination-background-color);
+  --calcite-pagination-arrow-text-color-hover: var(--calcite-color-brand);
+  --calcite-pagination-arrow-text-color: var(--calcite-pagination-text-color);
 }
 
 :host([scale="s"]) {
+  --calcite-internal-pagination-item-size: var(--calcite-size-xxl);
+  --calcite-internal-pagination-item-space: var(--calcite-size-xxs);
+
   & .chevron,
   & .page,
   & .ellipsis {
-    @apply text-n2h h-6 px-1;
-    min-inline-size: theme("width.6");
+    @apply text-n2h;
   }
 }
 
 :host([scale="m"]) {
+  --calcite-internal-pagination-item-size: var(--calcite-size-xxxl);
+  --calcite-internal-pagination-item-space: var(--calcite-size-sm);
+
   & .chevron,
   & .page,
   & .ellipsis {
-    @apply text-n1h h-8 px-2;
-    min-inline-size: theme("width.8");
+    @apply text-n1h;
   }
 }
 
 :host([scale="l"]) {
+  --calcite-internal-pagination-item-size: 44px;
```

Comment:
@alisonailea Seems like I can't get a variable with this value to work - directly importing the file that has the `size-44` reference doesn't seem to work, are any examples of these variables being used in component css files?

Reply:
you can do an `@import '~@esri/calcite-design-tokens/dist/scss/core.scss'`

Reply:
Updated and used the variable - can I get another look? Thanks